tests/test_template_types.py

```diff
@@ -838,9 +839,9 @@ def test_WithSubjectTemplate_character_count(content, values, expected_count):
     ("((placeholder))  ", {"placeholder": "  "}, None, 0),
     ("  ", {}, None, 0),
     ("Content with ((placeholder))", {"placeholder": "something extra"}, "GDS", 33),
-    ("Just content", {}, "GDS", 12),
-    ("((placeholder))  ", {"placeholder": "  "}, "GDS", 5),
-    ("  ", {}, "GDS", 5),
+    ("Just content", {}, "GDS", 17),
```

Comment:
This one meets my expectation of us sending the provider `GDS: Just content` and I don't see a code change in this commit that would change it from 12 characters to 17. Does that mean the test was failing before?

Reply:
I'm not sure I understand... "Just content" is 12 characters, and "GDS: " is 5 characters, so 17 characters together, no?

I just added tests for this case because it was not tested before.